firmware/robot2015/hw-test/attiny-spi-test.cpp

```diff
@@ -0,0 +1,72 @@
+#include "mbed.h"
+#include "KickerBoard.hpp"
+#include "pins-ctrl-2015.hpp"
+#include "SharedSPI.hpp"
+
```

Comment:
It would be good to put a comment up here giving a short description of what this program does.